docker/settings_docker.py

```diff
@@ -0,0 +1,64 @@
+import os
+from django.core.exceptions import ImproperlyConfigured
+import ast
+
+
+def get_env_variable(var_name):
+    msg = "Set the %s environment variable"
+    try:
+        return os.environ[var_name]
+    except KeyError:
+        error_msg = msg % var_name
+        raise ImproperlyConfigured(error_msg)
+
+
+def get_optional_env_variable(var_name):
+    try:
+        return os.environ[var_name]
+    except KeyError:
+        return None
+
+
+# options are either "PROD" or "DEV" (installing with Dev mode set gets you extra dependencies)
+MODE = get_env_variable("DJANGO_MODE")
+
+DEBUG = ast.literal_eval(get_env_variable("DJANGO_DEBUG"))
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.contrib.gis.db.backends.postgis",
+        "NAME": get_env_variable("PGDBNAME"),
+        "USER": get_env_variable("PGUSERNAME"),
+        "PASSWORD": get_env_variable("PGPASSWORD"),
```

Comment:
Can we add the options key from the main settings?

Reply:
This file's unused.  Some of this is setting up for local development with containers - this particular file will not be returning.